server/server.js

```diff
@@ -83,6 +74,6 @@ spdy
       return process.exit(1)
     } else {
       console.log('Listening on port: ' + port + '.')
-      verifyTask.start();
```

Comment:
Why start() method is here since its a cron functionality it should be in the services